Tests/UnsafeTriangulatorEditorTests.cs

```diff
@@ -239,13 +233,11 @@ public void UnsafeTriangulatorOutputStatusTest()
         [Test]
         public void UnsafeTriangulatorPlantHoleSeedsAutoTest()
         {
-            using var positions = LakeSuperiorPoints(Allocator.Persistent);
-            using var constraints = LakeSuperiorConstraints(Allocator.Persistent);
             var t = new UnsafeTriangulator<T>();
             var input = new LowLevel.Unsafe.InputData<T>()
             {
-                Positions = positions,
-                ConstraintEdges = constraints,
+                Positions = LakeSuperior.Points.Scale(1000, typeof(T) == typeof(int2)).DynamicCast<T>().AsNativeArray(),
+                ConstraintEdges = LakeSuperior.Constraints.AsNativeArray(),
```

Comment:
Fyi: may crash Unity due to https://github.com/andywiecko/BurstTriangulator/issues/182

Reply:
Not really, since this is `static readonly`. This will be fixed with #182 fixups.